end_to_end/test_gamma.sh

```diff
@@ -1,5 +1,8 @@
+#!/bin/bash
 set -e
 idx=$(date +%Y-%m-%d-%H-%M)
+run_dir="${1:-.}"
```

Comment:
Why? Why not just `cd run_dir && command` on the XLML side?

(If there is a good reason, happy to merge this!)

Reply:
Updated. Thank you!